### DISCUSSION TITLE
Install vs Acquisition numbers for extensions

### DISCUSSION THREAD

**Discussion by seemakamath** [microsoft/vscode-discussions#617] [Extension Development QnA] [answered]
When I look at the `acquisition` tab in my **Marketplace report**, the numbers are way higher than the `install` numbers shown in the **Marketplace** . I would expect `acquisitions` to be `uniques` and `installs` be `total downloads` across versions. So acquisitions < installs but I see the opposite.  I spent a bit of time searching online for answers so hoping someone here can help. How are the two numbers calculated? Where can I find more details on these numbers?

**Comment by alefragnani** [comment, marked as answer]
Hi @seemakamath ,

I remember that a couple of years ago there was some changes in the marketplace reports, but I didn't find the original post with the details, unfortunately. And the Marketplace Reports are being improved from time to time, based on extension authors feedback. 

If I remember correctly the `Installs` in the marketplace page had switched from _number of downloads_ to _number of active installs_. This means that if a user download/install an extension and uninstall it latter, it would be counted as _acquisition_ but removed from _installs_ latter. 

But, I guess the best place to look for feedback and a more precise information about the marketplace, is in the VS Marketplace repo (https://github.com/microsoft/vsmarketplace). It was created a couple of years ago with that in mind. I did a quick look and found a few issues that could help you, like https://github.com/microsoft/vsmarketplace/issues/4, https://github.com/microsoft/vsmarketplace/issues/353 and https://github.com/microsoft/vsmarketplace/issues/477.

Hope this helps

**Reply by alefragnani** [reply]
I just found the VS Code issue where the Dashboard Stats has been notified (https://github.com/microsoft/vscode/issues/94744), but not the detailed post....

**Reply by seemakamath** [reply]
@alefragnani Thanks so much!
